src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
```

Comment:
Consider using a more descriptive class name than `.UserInfo`. For example, `.user-info-container` or `.user-info-wrapper`. 🤔